tests/prov/test_prov_k8s_cortx_upgrade.py

```diff
@@ -372,3 +372,113 @@ def test_42179(self):
         assert_utils.assert_true(resp)
         self.collect_sb = False
         LOGGER.info("--------- Test Completed ---------")
+
+    @pytest.mark.lc
+    @pytest.mark.cortx_upgrade
+    @pytest.mark.tags("TEST-41951")
+    def test_41951(self):
+        """Verify suspend already suspended upgrade."""
+        pull_image_thread_list = []
+        LOGGER.info("Test Started.")
+        LOGGER.info("Step 1: Get installed version.")
+        installed_version = self.upgrade_obj.prov_obj.get_installed_version(
+            self.master_node_list[0])
+        resp = self.upgrade_obj.prov_obj.generate_and_compare_both_version(
+            self.upgrade_image, installed_version)
+        assert_utils.assert_true(resp)
```

Comment:
Its should be  assert_utils.assert_true(resp[0],resp[1])

Reply:
Here we are just capturing response to get installing version is higher than existing install version. If installed version is higher then it will return false then it will handle in assert

Reply:
As tuple will be returned from generate_and_compare_both_version, 
need to assert on resp[0]